src/components/Popups/TransactionPopup.tsx

```diff
@@ -13,15 +14,7 @@ const RowNoFlex = styled(AutoRow)`
   margin-right: 16px;
 `
 
-export default function TransactionPopup({
-  hash,
-  success,
-  summary,
-}: {
-  hash: string
-  success?: boolean
-  summary?: string
-}) {
+export default function TransactionPopup({ hash, success, summary }: PopupContent) {
```

Comment:
While we're removing default exports everywhere. It makes sense to slowly do it in PRs. :)

```suggestion
export function TransactionPopup({ hash, success, summary }: PopupContent) {
```

Reply:
Removed and adjusted imports.